build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
   // This dependency is found on compile classpath of this component and consumers.
   compile 'com.google.code.gson:gson:2.8.0'
   compile 'ca.uhn.hapi.fhir:hapi-fhir-base:3.3.0'
+  compile 'ca.uhn.hapi.fhir:hapi-fhir-structures-r4:3.3.0'
```

Comment:
No R4 yet. Please remove.

Reply:
Removing the r4 structures causes unit test to fail and I'm not sure why. The validator throws and error complaining that it doesn't have some r4 class, and the only information I could find online was this [github issue](https://github.com/jamesagnew/hapi-fhir/issues/957) which doesn't really help.